### PR TITLE
Create 2023-06-01

### PR DESCRIPTION
**Pull request by Mburnett-VA**
Veteran Service History & Eligibility V2 Release Notes

### CODE REVIEW

**Review by darduda — CHANGES_REQUESTED** [review]
reviewed with some very minor changes. The only major changes would be to update the links showing in the OAuth and CCG part since they point to the claims API right now.

File: content/verification/veteran-verification/release-notes/2023-06-01 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+As always, it's important to know that the new version must support backward compatibility. It should:
+
+- Be prepared to handle HTTP status codes not explicitly specified in endpoint definitions.
+
+- Be tolerant of unknown fields in the payload and ignore the deserialization of new fields.
+
+- Follow the redirect when the server returns an HTTP status code 301 (Moved Permanently).
+
+- Review the API documentation to learn more about the new version of this API.
```
[the remark this answers is not in the document]
Comment (Mburnett-VA) [reply content/verification/veteran-verification/release-notes/2023-06-01]: Done

**Review by joeabraun — CHANGES_REQUESTED** [review]
Just need to change the URL links for Auth Code Flow and CCG, and then should be good to go.

File: content/verification/veteran-verification/release-notes/2023-06-01.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+As always, it's important to know that the new version must support backward compatibility. It should:
+
+- Be prepared to handle HTTP status codes not explicitly specified in endpoint definitions.
+
+- Be tolerant of unknown fields in the payload and ignore the deserialization of new fields.
+
+- Follow the redirect when the server returns an HTTP status code 301 (Moved Permanently).
+
+- Review the [API documentation](https://developer.va.gov/explore/verification/docs/veteran_verification?version=current) to learn more about the new version of this API.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2023-06-01.md:35]:
```suggestion
Review the [API documentation](https://developer.va.gov/explore/verification/docs/veteran_verification?version=current) to learn more about the new version of this API.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]

**Review by joeabraun — APPROVED** [review]
This looks good to me. UX approves!

**Pull request merged**